src/solvers/Makefile

```diff
@@ -222,6 +231,28 @@ $(MINISAT2)/minisat/core/Solver$(OBJEXT): $(MINISAT2)/minisat/core/Solver.cc
 endif
 endif
 
+ifneq ($(MERGESAT),)
+ifeq ($(BUILD_ENV_),MSVC)
+sat/satcheck_minisat2$(OBJEXT): sat/satcheck_minisat2.cpp
```

Comment:
I would add a comment here that Mergesat is invoked from satcheck_minisat2.cpp

Reply:
Done!